web_app/package.json

```diff
@@ -2,13 +2,19 @@
   "name": "pitoco",
   "version": "0.0.1",
   "private": true,
+  "scripts": {
+    "start": "npm-run-all --parallel tailwind-watch shadow-watch",
+    "tailwind-watch": "tailwindcss -i tailwind/main.css -o resources/public/compiled/main.css --watch",
+    "shadow-watch": "shadow-cljs watch app"
+  },
   "devDependencies": {
-    "shadow-cljs": "2.15.9"
+    "shadow-cljs": "2.15.9",
+    "tailwindcss": "3.1.4",
+    "npm-run-all": "4.1.5"
   },
   "dependencies": {
-    "daisyui": "^1.14.0",
-    "npx": "^10.2.2",
```

Comment:
I removed `npx` because it is non-deterministic. It always get the latest version. This is a bad decision for many reasons: future breaking changes and malicious code execution.

Now we have an explicit `tailwindcss` version.